.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: python
 python: 3.6
 install: true
+cache:
+  directories:
+  - $HOME/.m2
 
 git:
   depth: false
   quiet: true
 
 stages:
   - name: tests
-    if: type IN (push, pull_request)
```

Comment:
why do we want to delete this ,do we want to run all the tests, all of the time ?
I think unit tests should be split from other tests and only be run on `push` and the other tests only on PR.
see also https://github.com/EGA-archive/LocalEGA/pull/25#discussion_r252970382

Reply:
Why not? I mean, it doesn't really matter to me, so I can bring it back. But again: why not?

Reply:
The only reason I could think of for keeping, is less conflicting PR merges ... e.g. it will be modified by this: https://github.com/EGA-archive/LocalEGA/pull/25/files#diff-354f30a63fb0907d4ad57269548329e3

Reply:
I propose to solve problems once they come. It's not a huge effort to merge two lines of code. But, as I said, this is not a major issue for me, so I can revert this change if we all think that it's better. Or it can be reverted in that #25 PR.

Reply:
ok, then we shall see when we merge.